pkg/cmd/server/bootstrappolicy/constants.go

```diff
@@ -40,6 +43,8 @@ const (
 	SelfProvisionerRoleName   = "self-provisioner"
 	BasicUserRoleName         = "basic-user"
 	StatusCheckerRoleName     = "cluster-status"
+	ImagePullerRoleName       = "image-puller"
```

Comment:
we've said (just today), that any role prefixed with "system:" is ours and we may choose to mutate it.  Is there any chance that we'll want to tweak these?